### PR TITLE
Add a 1.x => 2.x migration guide

### PR DESCRIPTION
**Pull request by GabeLambda**
Accounts for needed configuration changes for encryption updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,30 @@ Note that production usage should always create the user with a password. For ex
...
+item per registry entry
+
+General configuration will look like the existing configuration used for the registry (e.g. using `;` as a delimiter
+between items per registry entry).
+
+Full configuration may thus look something like this:
+
+    multi_tenant_key_registry=dict(
+         context_keys="client-id1,client-id2",
+         key_ids="key1a;key1b,key2a,key2b",
```
Comment (piercefreeman) [inline README.md:99]: I think you mean `key1a;key1b,key2a;key2b` here?

**Review by piercefreeman — APPROVED** [review]
LGTM - very helpful guide.

Small nit, I find semicolons to be a higher-order separator than a comma. So I would have expected `key1a;key1b,key2a;key2b` to parse into `[key1a] [key1b,key2a] [key2b]`. I suggested in another location that we use `&` for this scenario instead of semicolons. But since it's already implemented here and this is just a doc update I wouldn't block for that.

File: README.md (hunk 1 above)
Comment (GabeLambda) [reply README.md:99]: Yes, good eye.

**Comment by GabeLambda** [comment]
@piercefreeman microcosm generally accepts the usage of comma separated values for translating strings into lists, and most users of microcosm-related libraries should probably expect it. But I can see why it might be confusing.

**Pull request merged**